index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
```

Comment:
- [ ] Let's change this `<div>` tag to a `<form>` tag with the attributes `method="post"` and `action="<formspree url>"`.

This will allow the browser to send information posted by the users and it's the minimal information required for a form to work properly.

If you haven't do it yet, you can create an endpoint for this form at [Formspree](https://formspree.io/html) following the instructions in their webpage. You'll also be able to see the responses to your form there! How awesome is that? :tada: